_pages/install/browser-support.md

```diff
@@ -0,0 +1,55 @@
+
+The approach to browser support in Spartacus is as follows:
+
+- Angular is trusted to be compatible with evergreen browsers.
+- Other standards are trusted to be compatible with evergreen browsers (the [Can I Use?](http://caniuse.com/) website is a good example).
+- Spartacus has many automated quality checks that align the code with those standards (such as the TypeScript standard, and [Can I Use?](http://caniuse.com/)).
+- Spartacus end-to-end tests explicitly test only the Chrome browser.
+
+Nonetheless, with a bit of effort, it is still possible to get Spartacus to work with older browsers. The following are some of the steps you can take to get an older browsers to work with Spartacus:
+
+- Transpile the TypeScript into older versions of JavaScript.
+- Use polyfills to add JavaScript files that can provide some of the standard web features that are missing in some older browsers.
+- Use PostCSS to automatically add vendor prefix style rules when needed in an after process.
+
+## Working with IE11
+
+If your Spartacus storefront needs to support IE11, you must do the following:
+
+- Use JavaScript polyfills to add missing browser capabilities. The polyfills are prepared in a newly-rendered Angular CLI application, so it is typically only a matter of "uncommenting" some of these polyfills. For more information, see [Browser support](https://angular.io/guide/browser-support) in the official Angular documentation.
+- Customize the Spartacus styles, or bring in a custom style layer.
+
+The default Spartacus style library is not prepared for IE11. Since most customers will not use the Spartacus styles in the first place, it is a typical custom effort to build your styles in such a way that they support the required browsers.
+
+The following are some of the standard CSS features that are used in Spartacus styles, and that will not work for IE11:
+
+- Custom properties, which are used in Spartacus themes, and which are not supported in IE11. For more information, see the browser support tables for custom properties on the [Can I Use?](https://caniuse.com/?search=custom%20properties) website.
+- Logical properties, which are used for bidirectional text and layout, and which are not supported in IE11. For more information, see the browser support tables for logical properties on the [Can I Use?](https://caniuse.com/?search=logical%20properties) website.
+
+**Note:** The Angular framework that underlies Spartacus *does* support IE11. As a result, it is very likely that the application logic works flawlessly with IE11, and the limitations for IE11 are only related to the Spartacus styles. That being said, Spartacus is not actively tested with IE11.
```

Comment:
we could also add the notes about 3rd party libs here (the ones that might not work with ie11)